src/route/user-route.ts

```diff
@@ -0,0 +1,41 @@
+import { Router } from 'express';
+
+// eslint-disable-next-line import/no-cycle
+import { logger } from '../app';
+import { User } from '../entity/user';
+import { userToUserDTO } from '../dtos/user-dto';
+
+export const userRoute = Router();
+
+userRoute.get('/:id', async (req, res) => {
+    if (!req.params.id) {
+        logger.error(`Reequest parameter id is missing`);
+        res.status(400).send();
+        return;
+    }
+    const userID = req.params.id;
+    const user = await User.findOneBy({ id: userID });
+    if (!user) {
+        logger.error(`User with id ${req.params.id} not found`);
```

Comment:
log4j-style vulnerability due to unescaped injection of user controlled variable.